src/content/en/fundamentals/instant-and-offline/offline-cookbook/index.md

```diff
@@ -308,8 +308,6 @@ This code updates caches before showing a notification:
 
 <img src="images/cm-on-bg-sync.png">
 
-Dogfood: Background sync hasn't yet landed in Chrome stable.
```

Comment:
@TalAter - would you please also update the `{# wf_last_updated: xxx #}` field as well. We use that to ensure that translations are kept up to date.